cellrank/tl/kernels/_velocity_kernel.py

```diff
@@ -214,13 +213,19 @@ def compute_transition_matrix(
             )
             mode = VelocityMode.DETERMINISTIC
 
-        if mode == VelocityMode.STOCHASTIC and not hasattr(scheme, "hessian"):
-            logg.warning(
-                f"Unable to detect a method for Hessian computation. If using predefined functions, consider "
-                f"installing `jax` as `pip install jax jaxlib`.\n"
-                f"Defaulting to mode `{VelocityMode.MONTE_CARLO.s!r}`"
-            )
-            mode = VelocityMode.MONTE_CARLO
+        if mode == VelocityMode.STOCHASTIC:
+            try:
+                if not hasattr(
+                    scheme, "hessian"
+                ):  # this can also raise in our velocity scheme definition
+                    raise NotImplementedError
+            except NotImplementedError:
+                logg.warning(
+                    f"Unable to detect a method for Hessian computation. If using predefined functions, consider "
+                    f"installing `jax` as `pip install jax jaxlib`\n"
+                    f"Defaulting to `mode={VelocityMode.MONTE_CARLO.s!r}` and `n_samples={n_samples}`"
+                )
+                mode = VelocityMode.MONTE_CARLO
```

Comment:
I guess this fixed the Jax error I was experiencing?

Reply:
Yes.

Reply:
amazing, thanks.